tools/taskcluster/tasks.py

```diff
@@ -84,10 +84,11 @@ def craft_release_build_task(self, tag):
             ['secrets:get:project/mobile/firefox-tv/tokens'],
             {
                 'public': artifact('directory', '/opt/firefox-tv/app/build/outputs/apk')
-            }
+            },
+            chain_of_trust=True,  # Needed for sign and push task verification
         )
 
-    def _craft_shell_task(self, name, script, scopes, artifacts):
+    def _craft_shell_task(self, name, script, scopes, artifacts, *, chain_of_trust=False):
```

Comment:
What does a `*` parameter do? Is that a typo? I've only seen it used to unpack containers/keyword args like (`def function(*container)` (or whatever it's called).

Reply:
It's a neat python feature that forces parameters after the `*` to be named.
So, I can't do: `_craft_shell_task('name', script, [], artifacts, True)`
I have to do: `_craft_shell_task('name', script, [], artifacts, chain_of_trust=True)`